taichi/runtime/llvm/runtime.cpp

```diff
@@ -1036,6 +1036,18 @@ f32 cuda_shfl_up_sync_f32(u32 mask, f32 val, i32 delta, int width) {
   return 0;
 }
 
+bool cuda_all_sync(u32 mask, bool bit) {
+  return false;
+}
+
+int32 cuda_all_i32(int32 predicate) {
```

Comment:
Same question: Is this function redundant? It seems that `cuda_all_i32` can be replaced by `cuda_all_sync_i32`. If you want to keep this function, please expose it to python and add corresponding unit tests. :)

Reply:
I will remove `cuda_all_i32`.